Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/operator-framework/ansible-operator:v1.3.0
```

Comment:
Can we check if this is ubi-8 based?

Reply:
This is a ubuntu20.04 image

Reply:
That does not work, our images all have to be ubi-8 based.

Reply:
My bad, it is ubi based 
https://github.com/operator-framework/operator-sdk/blob/f8f3690d9a3198d2f63411ea2ac53b2bce0c3c9d/images/ansible-operator/Dockerfile#L1
Checked on the running operator too

bash-4.4$ cat /etc/os-release 
NAME="Red Hat Enterprise Linux"
VERSION="8.3 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.3"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.3 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8.3:GA"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.3
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.3"